.gitignore

```diff
@@ -1,7 +1,7 @@
 example.json
 crash.log
 packer-builder-softlayer
-
+vendor
```

Comment:
- [ ] This ignores any file called "vendor".  
To ignore the top-level _vendor_ directory, state it more specifically:  
```
/vendor/
```